Lib/fontTools/feaLib/builder.py

```diff
@@ -647,6 +657,11 @@ def buildLookups_(self, tag):
             if lookup.table != tag:
                 continue
             lookup.lookup_index = len(lookups)
+            self.lookup_locations[tag][len(lookups)] = [
```

Comment:
this looks more readable 
```suggestion
            self.lookup_locations[tag][lookup.lookup_index] = [
```